st2api/st2api/controllers/v1/actionexecutions.py

```diff
@@ -153,7 +155,16 @@ def _schedule_execution(self, liveaction):
             liveaction_db.parameters = param_utils.render_live_params(
                 runnertype_db.runner_parameters, action_db.parameters, liveaction_db.parameters,
                 liveaction_db.context)
-        except ParamException as e:
+        except ParamException:
+            LOG.info('ParamException')
```

Comment:
Hm, this log message is not really useful (and it doesn't include traceback or anything).

Should probably be LOG.warn with a better error message and traceback?